autotst/calculator/vibrational_analysis.py

```diff
@@ -188,6 +188,10 @@ def obtain_percent_changes(self):
             if bond.reaction_center:
                 labels = sorted([self.ts.rmg_molecule.atoms[i].label.lstrip('*'), self.ts.rmg_molecule.atoms[j].label.lstrip('*')])
                 key = f'd{labels[0]}{labels[1]}'
+                if key == 'd24':
+                    key = 'd12'
+                elif key == 'd14':
+                    key = 'd23'
```

Comment:
do we know why this might be happening for disproportionation?

Reply:
This is just the way that it's always been coded for AutoTST. This is how the labeling works in `reaction.py` for disproportionation at the moment:
```
            lbl1 = self.rmg_molecule.get_all_labeled_atoms()["*2"].sorting_label
            lbl2 = self.rmg_molecule.get_all_labeled_atoms()["*4"].sorting_label
            lbl3 = self.rmg_molecule.get_all_labeled_atoms()["*1"].sorting_label
```

So `d12` corresponds between `lbl1` and `lbl2` which are atoms labeled `*2` and `*4` respectively.